src/test/java/elizCurtnazar/tests/tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,118 @@
+package elizCurtnazar.tests.tema6;
+
+import elizCurtnazar.driver.BrowserManager;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HerokuAlerts {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToHerokuHomePage();
+        openAlertsPage();
+        openFirstAlert();
+        acceptFirstAlert();
+        openSecondAlert();
+        acceptAlert();
+        openThirdAlert();
+        acceptThirdAlert();
+        try {
+            openFakeAlertsPage();
+            System.out.println("Am intrat in blocul de try.");
+
+        } catch (NoSuchElementException e) {
+            System.out.println("Am intrat pe blocul de catch");
+            File file = driver.getScreenshotAs(OutputType.FILE);
```

Comment:
Use FileUtils.takeScreenshot method instead of these lines to avoid code duplication. Thanks!

Reply:
done